doc/introduction.rst

```diff
@@ -103,6 +131,10 @@ examples (for code illustrating some of these use-cases, see the
   the data into a new embedding space before feeding it into another machine
   learning algorithm.
 
+The most common use-case of metric-learn would be to learn a Mahalanobis metric,
+then transform the data to the learned space, and then resolve one of the task
+above.
```

Comment:
this is confusing, and somehow make it seems like bilinear similarity is not useful. It is also redundant with the last item of the previous list. I would just remove but modify slightly the last item (which refers to matrix L of Mahalanobis):

"More generally with Mahalanobis distances, the learned transformation L can be used to project the data into a new embedding space before feeding it into another machine learning algorithm."

Reply:
Removed the redundant paragraph and changed the last item of the list.